src/shape/plane.js

```diff
@@ -7,12 +7,12 @@ Object.assign(pc, function () {
      * @name pc.Plane
      * @classdesc An infinite plane.
      * @description Create an infinite plane.
-     * @param {pc.Vec3} [point] - Point position on the plane. The constructor takes a reference of this parameter.
-     * @param {pc.Vec3} [normal] - Normal of the plane. The constructor takes a reference of this parameter.
+     * @param {pc.Vec3} [point] - Point position on the plane. The constructor takes a copy of this parameter.
+     * @param {pc.Vec3} [normal] - Normal of the plane. The constructor takes a copy of this parameter.
      */
     var Plane = function Plane(point, normal) {
-        this.normal = normal || new pc.Vec3(0, 0, 1);
-        this.point = point || new pc.Vec3(0, 0, 0);
+        this.normal = normal ? normal.clone() : new pc.Vec3(0, 0, 1);
```

Comment:
Also potentially breaking

Reply:
good point, moving these changes to separate PR